src/test/java/school/faang/user_service/service/SkillServiceTest.java

```diff
@@ -0,0 +1,182 @@
+package school.faang.user_service.service;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import school.faang.user_service.dto.skill.SkillCandidateDto;
+import school.faang.user_service.dto.skill.SkillCreateDto;
+import school.faang.user_service.dto.skill.SkillDto;
+import school.faang.user_service.entity.Skill;
+import school.faang.user_service.entity.User;
+import school.faang.user_service.entity.recommendation.Recommendation;
+import school.faang.user_service.entity.recommendation.SkillOffer;
+import school.faang.user_service.execption.DataValidationException;
+import school.faang.user_service.mapper.SkillCandidateMapper;
+import school.faang.user_service.mapper.SkillCreateMapper;
+import school.faang.user_service.mapper.SkillMapper;
+import school.faang.user_service.repository.SkillRepository;
+import school.faang.user_service.repository.UserRepository;
+import school.faang.user_service.repository.recommendation.SkillOfferRepository;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@ExtendWith(MockitoExtension.class)
+public class SkillServiceTest {
+    private static final long USER_ID = 1L;
+    private static final long SKILL_ID = 1L;
+
+    @Mock
+    private SkillRepository skillRepository;
+
+    @Mock
+    private SkillOfferRepository skillOfferRepository;
+
+    @Mock
+    private UserRepository userRepository;
+
+    @Mock
+    private SkillMapper skillMapper;
```

Comment:
Лучше добавить аннотацию Spy, тогда mapper будет реально выполнять свою работу, а не будет заглушкой
```suggestion
    @Spy
    private SkillMapperImpl skillMapper;
```

Reply:
Всему свое время